ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -125,8 +125,17 @@ function is_openshift_version_gte() {
     printf '%s\n%s' "$1" "${DS_OPENSHIFT_VERSION}" | sort -C -V
 }
 
+function extract_oc_for_upgrade() {
+    extract_dir=$(mktemp --tmpdir -d "installer-upgrade-XXXXXXXXXX")
+    oc adm release extract --registry-config ${DS_WORKING_DIR}/pull_secret.json --command=oc --to "${extract_dir}" "$1"
```

Comment:
Seems like it complains about the pull secret
```
error: unable to load --registry-config: stat /opt/dev-scripts/pull_secret.json: no such file or directory
```
@derekhiggins @elfosardo do you have any ideas on how I should handle this?